tests/unit/objects/test_preferences.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+import unittest
+
+from quickbooks import QuickBooks
+from quickbooks.objects.preferences import Preferences
+
+
+def check_valid_date(date_text):
+    try:
+      datetime.datetime.strptime(date_text, '%Y-%m-%d')
+      return True
+    except ValueError:
+      raise ValueError("Incorrect data format, should be YYYY-MM-DD")
+
+
+class PreferencesTests(unittest.TestCase):
```

Comment:
Are we using pytest functional tests going forward instead of unittest.TestCase class based tests?

Reply:
Yeah I'd like to. Not sure if I should make that change in this PR or commit to move them all over in another?